app/bundles/PageBundle/Controller/PageController.php

```diff
@@ -882,6 +882,7 @@ public function abtestAction($objectId)
             $clone->setVariantParent($entity);
         }
 
+        /** @phpstan-ignore-next-line */
```

Comment:
Some weird error that the variable isn't set, while it's cloned a bit further up. Probably a PHPStan bug, just ignoring here